Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fmt test stage
+.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fix-imports fmt test stage
```

```suggestion
.PHONY: all pull fetch patch generate clean codegen mod docs move-other patch-post fmt test stage
```
Is fix-imports from another commit or branch? Doesn't seem to refer to a section of this makefile.

---

Ooooohhh, yeah missed that.  The `fix-import` task was needed for the custom template that I was using earlier and was deleted alongside file change.

---

Updated